Homeworks/Homework W4-OOP_Exercises/Calculator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+namespace Homework_W4_OOP_Exercises
+{
+	public class Calculator
+	{
+	
+		public int Add(int x,int y)
+		{
+			int sum = x + y;
+			return sum;
+		}
+		public int Substract( int x, int y)
+		{
+			int sub;
+
+            if (x>y)
+			{
+                sub = x - y;
+            }
+			else
+			{
+				sub = y - x;
+			}
+			
+			return sub;
```

Comment:
Here the result can be wrong because if you have a calculator and do something like 5 - 10 the result is -5. But in your implementation, the result is 5.